pkg/sql/schemachanger/dml_injection_test.go

```diff
@@ -523,6 +537,12 @@ func TestAlterTableDMLInjection(t *testing.T) {
 								require.Subset(t, expectedResults, actualResults, errorMessage)
 								require.Subset(t, actualResults, expectedResults, errorMessage)
 
+								// If a query is provided, run it without checking the results—just
+								// ensure it doesn't fail.
+								if tc.query != "" {
+									_ = sqlDB.QueryStr(t, tc.query)
```

Comment:
super nit: this could use `Exec` instead of `QueryStr` since we don't care about the results.